### DISCUSSION TITLE
Configure Buildah to ignore xattr related errors

### DISCUSSION THREAD

**Discussion by teojgo** [containers/buildah#4465] [Q&A] [answered]
Is there a way to configure/build Buildah to ignore the xattr errors that might be thrown when performing a COPY/ADD step on a Containerfile? I see that there is the [IgnoreXattrErrors](https://github.com/containers/buildah/blob/3c94e5345231eba4c6e91d843adbf1e4e956c436/copier/copier.go#L343) boolean but I don't know how I could enforce it either when building or when running Buildah.

**Comment by flouthoc** [comment, marked as answer]
Hi @teojgo , Afaik there is no frontend options to configure this option so buildah users can't directly control it but if you have a use-case I think you should open a new feature request and we can discuss this and if all maintainers agree then this flag can be exposed :) . So I'd request you to create a new issue(RFE) with description and use-case :smile: 

cc @nalind @rhatdan @containers/buildah-maintainers
